src/components/Categories/GetCategories.vue

```diff
@@ -1,55 +1,58 @@
-<template>     
-    <div class = "container">  
-		<h2 :style="{textAlign:'center', marginTop:'30px', marginBottom:'30px'}">Danh sách tác giả</h2>
-		<table class="table">  
-			<thead>  
-				<th :style="{paddingLeft:'8px'}">Id</th>  
-				<th :style="{paddingLeft:'8px'}">Thể loại</th>  
-				<th :style="{paddingLeft:'8px'}">Mô tả</th>  
-				<th></th>  
-			</thead>  
-			<tbody>  
-				<tr v-for="item in categories" :key="item.id">  
-					<td>{{ item.id }}</td>  
-					<td>{{ item.name }}</td>  
-					<td>{{ item.description }}</td>  
-					<td><input class="btn btn-success" type='button' value='Chi tiết' @click="detail(item.id)"/></td>
-				</tr>  
-			</tbody>  
-		</table>  
-    </div>  
+<template>
+  <Card>
+    <template #title>
+      <div style="text-align: center">Danh sách thể loại</div>
+    </template>
+    <template #content>
+      <DataTable :value="categories">
```

Comment:
```suggestion
      <DataTable :value="categories" :autoLayout="true">
```

Và bỏ qua mấy style width cho từng cột bên dưới

Reply:
done